spec/unit/network/http/connection_spec.rb

```diff
@@ -42,31 +42,18 @@
     end
   end
 
-  context "when methods that accept a block are called with a block" do
-    let (:host) { "my_server" }
+  context "when methods that accept a block are called with a block", :vcr => true do
```

Comment:
Per https://www.relishapp.com/vcr/vcr/v/2-9-3/docs/test-frameworks/usage-with-rspec-metadata, with rspec 3 this can be just `:vcr` rather than `:vcr => true`. (It's a bit of a nit, but it seems to be the pattern that shows up in the reference docs, so may seem less different to vcr newbies like myself who go poking around on relishapp.)